src/utility/index.jsx

```diff
@@ -713,3 +713,79 @@ export function groupByArray(xs, key) {
 
 export const sumValues = f => Object.values(f).reduce((a, b) => a + b);
 
+/* eslint-disable camelcase */
+// https://dota2.gamepedia.com/Attributes
+export function compileLevelOneStats(hero) {
+  const statsBonuses = {
```

Comment:
fine with this but maybe we can fetch this automatically from the patch notes (whenever valve decides to change the bonuses again)?

cc @howardchung

Reply:
yeah idk if that's possible

Reply:
Alright